client/my-sites/checkout/checkout-thank-you/style.scss

```diff
@@ -881,6 +881,46 @@ main.akismet-checkout-thank-you {
 		}
 	}
 
+	.akismet-checkout-thank-you__api-key-header {
+		font-weight: 700;
+		font-size: 1.25rem;
+		line-height: 1.2;
+		margin-top: 1.5rem;
+		margin-bottom: 0.5rem;
+	}
+
+	.akismet-checkout-thank-you__key-clipboard {
+		display: flex;
+		margin-top: 16px;
+		max-width: 500px;
+		margin-bottom: 2rem;
+
+		input.form-text-input.akismet-checkout-thank-you__key-clipboard-input {
+			font-size: 1.5rem;
+			margin-right: 16px;
+			margin-left: 0;
+			background-color: var(--studio-gray-10);
+			color: var(--color-text);
+			border: none;
+		}
+
+		.button {
+			min-width: 100px;
+			font-size: 1.25rem;
+		}
+
+		@include breakpoint-deprecated( ">480px" ) {
```

Comment:
I see that the deprecated breakpoint is being used in this file quite a bit, but it looks like this value is the same as if you did this
```scss
@include break-mobile;
```

Reply:
Good catch - thanks!